x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/policy_list/policy_table/policy_table.tsx

```diff
@@ -327,36 +327,30 @@ export const PolicyTable: React.FunctionComponent<Props> = ({
     ) : (
       undefined
     ),
-    toolsRight: (
-      <EuiFlexGroup gutterSize="m" justifyContent="spaceAround">
-        <EuiFlexItem>
-          <EuiButton
-            color="secondary"
-            iconType="refresh"
-            onClick={reload}
-            data-test-subj="reloadButton"
-          >
-            <FormattedMessage
-              id="xpack.snapshotRestore.policyList.table.reloadPoliciesButton"
-              defaultMessage="Reload"
-            />
-          </EuiButton>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiButton
-            href={linkToAddPolicy()}
-            fill
-            iconType="plusInCircle"
-            data-test-subj="createPolicyButton"
-          >
-            <FormattedMessage
-              id="xpack.snapshotRestore.policyList.table.addPolicyButton"
-              defaultMessage="Create a policy"
-            />
-          </EuiButton>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    ),
+    toolsRight: [
```

Comment:
Can you add a `key` prop to each button in this array? That should clear up the react warnings I'm seeing in the browser console.

For more info, see: https://reactjs.org/docs/lists-and-keys.html

Reply:
Fixed.